spec/serializable/serializer_spec.rb

```diff
@@ -198,6 +198,22 @@ def custom_attributes
           ]
         }
       end
+
+      it "includes a whitelist of attributes if specified" do
+        serializer.as_json(person, { attribute_whitelist: [:name, :description] }).should == {
+          name: 'Gavin',
+          description: 'This is person #123',
+          custom_key: 'custom value for model id 123'
+        }
+      end
+
+      it "excludes a blacklist of attributes if specified" do
+        serializer.as_json(person, { attribute_blacklist: [:name, :description] }).should == {
+          id: '123',
+          href: '/people/123',
+          custom_key: 'custom value for model id 123'
```

Comment:
The interaction between custom attributes and the black/white lists feels a little magical. Again, worth ensuring an attribute can't be on both the custom and black lists? And ensuring it must be on the white list if there is a white list? (Feel free to ignore if this is adding too much unwanted complexity)